src/pages/index.js

```diff
@@ -1,45 +1,354 @@
-import appConfig from 'appConfig';
 import Head from 'next/head';
+import React, { useEffect, useState } from 'react';
```

Comment:
Can you please comment inline in this file to point out any changes that might be here that are not just copying over from the old app.js?

Reply:
Looking at the first commit in isolation is pretty helpful for this